src/System.Net.Primitives/tests/FunctionalTests/IPAddressParsing.cs

```diff
@@ -351,10 +351,11 @@ private static void ParseInvalidAddress(string invalidAddress, bool hasInnerSock
         [Fact]
         public void Parse_Null_Throws()
         {
-            Assert.Throws<ArgumentNullException>(() => { IPAddress.Parse(null); });
+            string test = null;
+            Assert.Throws<ArgumentNullException>(() => { IPAddress.Parse(test); });
 
             IPAddress ipAddress;
-            Assert.False(IPAddress.TryParse(null, out ipAddress));
+            Assert.False(IPAddress.TryParse(test, out ipAddress));
```

Comment:
The previous behaviour failed to build with ambigous overload error. Forcing the null to be of string type eliminates the Span version from the available candidates.